java/ql/lib/ext/java.util.stream.model.yml

```diff
@@ -87,3 +87,9 @@ extensions:
       - ["java.util.stream", "Stream", True, "takeWhile", "(Predicate)", "", "Argument[-1].Element", "ReturnValue.Element", "value", "manual"]
       - ["java.util.stream", "Stream", True, "toArray", "", "", "Argument[-1].Element", "ReturnValue.ArrayElement", "value", "manual"]
       - ["java.util.stream", "Stream", True, "toList", "()", "", "Argument[-1].Element", "ReturnValue.Element", "value", "manual"]
+
+  - addsTo:
+      pack: codeql/java-all
+      extensible: extNeutralModel
+    data:
+      - ["java.util.stream", "Collectors", "toList", "()", "manual"]
```

Comment:
You've excluded parameterless constructors from model consideration - you could do the same for parameterless static methods. And once we have that, I guess we should probably just drop this neutral model?

Reply:
I've opened a PR for this: https://github.com/github/codeql/pull/11717
I believe the neutral models for the below APIs will need to be dropped as well:
- `java.lang.System#currentTimeMillis()`
- `java.util.Collections#emptyList()`
- `java.util.Optional#empty()`
- `java.util.UUID#randomUUID()`